Sources/Views/Cells/MessageContentCell.swift

```diff
@@ -151,10 +157,22 @@ open class MessageContentCell: MessageCollectionViewCell {
         let topMessageLabelText = dataSource.messageTopLabelAttributedText(for: message, at: indexPath)
         let bottomMessageLabelText = dataSource.messageBottomLabelAttributedText(for: message, at: indexPath)
 
+        let sentDate = message.sentDate
+        let sentDateString = MessageKitDateFormatter.shared.string(from: sentDate)
+        let timeLabelFont : UIFont = .boldSystemFont(ofSize: 10)
+        let timeLabelColor: UIColor = .darkGray
```

Comment:
Just checking - dark mode / light mode look good? Can we add both screenshots to the PR description please?

Reply:
It got better now, Thanks